subscription.go

```diff
@@ -2,19 +2,36 @@ package pubsub
 
 import (
 	"context"
+	"github.com/libp2p/go-libp2p-core/peer"
+)
+
+type EventType uint8
```

Comment:
just `int`.

Reply:
sure, any particular reason why so I know for future enums?